packages/cursorless-engine/src/typings/Types.ts

```diff
@@ -1,14 +1,8 @@
-import type {
-  Range,
-  Selection,
-  TextDocument,
-  TextEditor,
-} from "@cursorless/common";
+import type { Range, Selection, TextEditor } from "@cursorless/common";
 import type { SyntaxNode } from "web-tree-sitter";
 
-export interface ProcessedTargetsContext {
-  getNodeAtLocation: (document: TextDocument, range: Range) => SyntaxNode;
-}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ProcessedTargetsContext {}
```

Comment:
leaving this here as empty because removing this touches a bunch of lines, so is cleaner as its own PR; see #1477